packages/core/echo/echo-schema/src/mutable/manipulation.ts

```diff
@@ -50,3 +52,13 @@ export const updateFieldNameInSchema = (
     ),
   );
 };
+
+export const updateTypenameInSchema = (schema: S.Schema<any>, typename: string): S.Schema<any> => {
```

Comment:
I see you following the convention here, but we should find another way to namespace these methods otherwise this suffix is becoming clumsy.

But it should be `setTypename(InSchema)` not update i think.

@dmaretskyi should we consider namespaces for packages with lots of public definitions? If these are internal, then they should be marked `@internal`.